vent/io/devices/sensors.py

```diff
@@ -284,9 +284,10 @@ def _convert(self, raw) -> float:
         raw = super()._convert(raw)
         fit_param = 2.5837e-05
         if(raw >= 0):
-            converted_flow = (-1.0*np.sqrt(raw)/np.sqrt(fit_param))
+            #converted_flow = (-1.0*np.sqrt(raw)/np.sqrt(fit_param))
+            converted_flow = 192.6426*(raw)**(1/1.9128)
         else:
-            converted_flow = (1.0*np.sqrt(-1.0*raw)/np.sqrt(fit_param))
+            converted_flow = -192.6426*(np.abs(raw))**(1/1.9128)

```

Comment:
and here is why we could not calculate VTE for a month #facepalm

Reply:
Yeah, it was only 2 weeks, actually! This is Sophie's calibration, replacing the old calibration file.